### PR TITLE
Support WAITFOR TIME/DELAY

### PR DESCRIPTION
**Pull request by RIC06X**
### Description

Previously Babelfish does not support TSQL WAITFOR statement and customers have to use workaround to make delay. This commit support TSQL WAITFOR TIME and WAITFOR DELAY. 

### Issues Resolved

WAITFOR TIME and WAITFOR DELAY supported.
Task: BABEL-122

### Test Scenarios Covered ###
- WAITFOR DELAY with constant time
- WAITFOR DELAY with variable
- WAITFOR DELAY within a procedure
- WAITFOR TIME with variable
- WAITFOR TIME within a procedure

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: contrib/babelfishpg_tsql/src/tsqlIface.cpp [hunk 1]
```diff
@@ -3984,7 +3984,27 @@ makeTryCatchStmt(TSqlParser::Try_catch_statementContext *ctx)
...
+	std::string query;
+
+    result->cmd_type = PLTSQL_STMT_EXEC;
+    result->lineno = getLineNo(ctx);;
+    result->is_call = true;
+    result->return_code_dno = -1;
+    result->paramno = 0;
+    result->params = NIL;
+
+	query += "EXEC ";
```
Comment (forestkeeper) [inline contrib/babelfishpg_tsql/src/tsqlIface.cpp]: spaces not right

File: test/JDBC/upgrade/latest/schedule [hunk 2]
```diff
@@ -442,3 +442,4 @@ datetimeoffset-timezone
 BABEL-4046
 host_id
 linked_srv_4229
+waitfor
```
Comment (forestkeeper) [inline test/JDBC/upgrade/latest/schedule:445]: add the waitfor tests into other schedule files

File: contrib/babelfishpg_tsql/src/tsqlIface.cpp (hunk 1 above)
Comment (RIC06X) [reply contrib/babelfishpg_tsql/src/tsqlIface.cpp]: fixed

**Review by forestkeeper — APPROVED** [review]
LGTM

**Pull request merged**